components/calendar/calendar-schedule-summary.tsx

```
@@ -0,0 +1,329 @@
+  const { events: allEvents, isLoading } = useAllCalendarEvents();
+
+  // Calculate date range based on view mode and offset
+  const { startDate, endDate, rangeLabel } = useMemo(() => {
+    const today = new Date();
+    let start: Date;
+    let end: Date;
+    let label: string;
+
+    switch (viewMode) {
+      case "today":
+        const targetDay = addDays(today, offset);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)